test/functional/feature_fee_estimation.py

```diff
@@ -73,36 +73,38 @@ def small_txpuzzle_randfee(
 
 def check_raw_estimates(node, fees_seen):
     """Call estimaterawfee and verify that the estimates meet certain invariants."""
-
-    delta = 1.0e-6  # account for rounding error
+    delta = Decimal('0.000001')  # account for rounding error
     for i in range(1, 26):
         for _, e in node.estimaterawfee(i).items():
-            feerate = float(e["feerate"])
+            feerate = Decimal(e["feerate"])
             assert_greater_than(feerate, 0)
 
-            if feerate + delta < min(fees_seen) or feerate - delta > max(fees_seen):
+            min_fees_seen = Decimal(min(fees_seen)).quantize(Decimal('0.00000001'))
+            max_fees_seen = Decimal(max(fees_seen)).quantize(Decimal('0.00000001'))
```

Comment:
We can't use `satoshi_round()` here because it will by default round down and that's not what's needed here? Wondering if that function can also provide an option to not round at all.

Reply:
For this function, I wasn't sure which rounding type to use. That's why I opted to keep the implementation and just change to Decimal